wiki/machine-learning/nlp_for_robotics.md

```diff
@@ -0,0 +1,200 @@
+---
```

Comment:
@sudo-prakhar This is a new page and requires an addition to the `navigation.yml`. You can take a look at #139 for an example of the changes required.